misc/tcb.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@PREFIX@
+exec_prefix=${prefix}
```

Comment:
What's the benefit of exporting `prefix` and `exec_prefix`?

Reply:
There are some versions of pkgconfig around, which require those two to be exported even if they are of no use.  For that reason I've added them.